File: data/src/main/java/ru/timekeeper/data/network/model/groupWallRemote/Reposts.java
package ru.timekeeper.data.network.model.groupWallRemote;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Reposts {

    @SerializedName("count")
    @Expose
    public Integer count;
    @SerializedName("user_reposted")
    @Expose
    public Integer userReposted;

}

File: data/src/main/java/ru/timekeeper/data/network/model/groupWallRemote/Reposts.kt
package ru.timekeeper.data.network.model.groupWallRemote

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class Reposts (

    @SerializedName("count")
    @Expose
    val count: Int? = null,
    @SerializedName("user_reposted")
    @Expose
    val userReposted: Int? = null

)
